velox/connectors/hive/HiveConnectorSplit.cpp

```diff
@@ -57,6 +57,20 @@ folly::dynamic HiveConnectorSplit::serialize() const {
       ? folly::dynamic(tableBucketNumber.value())
       : nullptr;
 
+  if (bucketConversion.has_value()) {
+    folly::dynamic bucketConversionObj = folly::dynamic::object;
```

Comment:
```
nit: folly::dynamic bucketConversionObj;
```

Reply:
```
folly::dynamic <name> = folly::dynamic::object;
folly::dynamic <name> = folly::dynamic::array;
```

This seems to be a common practice in this file. Am I missing something?

https://github.com/facebookincubator/velox/blob/213e44ff8432251c938d26fc1f552bd29e349439/velox/connectors/hive/HiveConnectorSplit.cpp#L38-L114